### DISCUSSION TITLE
为什么每次登录网页需要重新设置模型，能不能设置一次记住当前的模型呢，因为我想共享给别人使用

### DISCUSSION THREAD

**Discussion by qq9797444** [ChatGPTNextWeb/NextChat#5793] [General]
为什么每次登录网页需要重新设置模型，能不能设置一次记住当前的模型呢，因为我想共享给别人使用

**Comment by imoisture** [comment]
### 同问，同样的需求 😊

**Comment by noctiro** [comment]
你可以设置环境变量 `DEFAULT_MODEL` 来设置用户第一次打开网页时候的默认模型

如果你不想用户切换到其他模型，你可以设置 `CUSTOM_MODELS` 变量来删除你不想用的自带模型 详见 https://github.com/ChatGPTNextWeb/NextChat?tab=readme-ov-file#custom_models-optional
